src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -65,12 +41,27 @@ public async Task<ActionResult> GetUsersByOrganisationId(Guid organisationId)
         }
 
         [HttpPost]
-        public ActionResult CreateUser(Guid organisationId, OrganisationUserViewModel userViewModel)
+        public async Task<ActionResult> CreateUserAsync(Guid organisationId, CreateUserRequestViewModel viewModel)
         {
-            userViewModel.UserId = Guid.NewGuid().ToString();
-            userViewModel.OrganisationId = organisationId;
+            if (viewModel is null)
```

Comment:
should probably be consistent with
_usersRepository = usersRepository ?? throw new ArgumentNullException(nameof(usersRepository));
above

Reply:
This one is slightly different as I am not assigning it & to be honest it was what Resharper generated. Unfortunately it errors when I try the example.